.gitignore

```diff
@@ -23,3 +23,7 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+
+# IDE / Code Editor configs
+/.idea
```

Comment:
Was this added on purpose? Usually environment specific things like developers' IDEs make more sense in a [global `.gitignore`](https://stackoverflow.com/questions/7335420/global-git-ignore#7335487)